src/System.Private.CoreLib/shared/System/IO/Path.cs

```diff
@@ -812,8 +812,12 @@ public static string GetRelativePath(string relativeTo, string path)
 
         private static string GetRelativePath(string relativeTo, string path, StringComparison comparisonType)
         {
-            if (string.IsNullOrEmpty(relativeTo)) throw new ArgumentNullException(nameof(relativeTo));
-            if (PathInternal.IsEffectivelyEmpty(path.AsSpan())) throw new ArgumentNullException(nameof(path));
+            if (relativeTo == null)
+                throw new ArgumentNullException(nameof(relativeTo));
+            if (relativeTo.Length == 0)
+                throw new ArgumentException(nameof(relativeTo));
```

Comment:
Does this need an error message that describes what is wrong with the argument?

Reply:
Should I add `SR.Arg_RelativePathEmpty` with vakue: `The relative path is empty.`?

Reply:
I think the existing `Arg_PathEmpty` is good enough. Note that the exception message is also going to contain the argument name that will say which exact argument is empty.